README.md

```diff
@@ -65,38 +65,52 @@ NOTE: OpenCL host side program should be linked with icd loader `-lOpenCL`.
 ```
 $ <pocl-install-dir>/bin/poclcc -l
 
+// The following output should be shown:
 LIST OF DEVICES:
 0:
   Vendor:   THU
     Name:   Ventus GPGPU device
  Version:   2.2 HSTR: THU-ventus-gpgpu
 ```
 
-Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example(Work in progress).
+Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example:
+```
+$ ./<pocl-install-dir>/examples/example1
```

Comment:
./<pocl-install-dir>/examples/example1/exapmple1

Reply:
Thanks for pointing it out!